# JOMG-Henry_Trousdell_Cyrill_Et_al
Characterization of gene expression signatures for the identification of cellular heterogeneity in the developing mammary gland 

Characterization of gene expression signatures for the identification of cellular
heterogeneity in the developing mammary gland
Samantha Henry1, 2, Marygrace C. Trousdell1, Samantha L. Cyrill1, Yixin Zhao1, Mary. J.
Feigman1, Julia M.Bouhuis3, Dominik A. Aylard4, Adam Siepel1, Camila O. dos Santos1a
1Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724. 2Graduate Program in Genetics,
Stony Brook University.3Erasmus Medical Center in Rotterdam, the Netherlands. 5College of
Biological Sciences, University of California, Davis.
Correspondence and requests for materials should be addressed to C.O.D.S (email:
dossanto@cshl.edu)

The developing mammary gland utilizes several transcription-dependent networks to
support multiple cellular identities and differentiation pathways. Recent technological
advancements that allow for single-cell profiling of gene expression have provided an initial
picture into the cellular heterogeneity within the mammary epithelium across the diverse stages
of gland maturation. Still, a deeper dive into extended molecular signatures of all mammary
resident cells would improve our understanding of the diversity of mammary epithelial and non-
epithelial cellular populations across different tissue developmental stages, mouse strains and
mammalian species. Here, we combined differential mammary gland fractionation and
transcriptional profiles obtained from FACS-isolated mammary cells to improve our definitions of
mammary-resident, cellular identities at the single-cell level. Our approach yielded a series of
expression signatures that illustrate the heterogeneity of mammary epithelial cells, specifically
those of the luminal fate, and uncovered transcriptional changes to their lineage-defined, cellular
states that are induced during gland development. Our analysis also provided molecular
signatures that identified non-epithelial mammary cells, including those of adipocytes, fibroblasts
and not so abundant immune cells. Lastly, we extended our study to elucidate expression
signatures of human, breast-resident cell fates, a strategy that allowed for the cross-species
comparison of mammary epithelial identities. Collectively, our approach improved the existing
signatures of normal mammary epithelial cells, as well as elucidated the diversity of non-epithelial
cells in murine and human breast tissue. Our study provides a useful resource for future studies
that use single-cell molecular profiling strategies to understand normal and malignant breast
development.
